shared/teams/delete-team/container.tsx

```diff
@@ -5,12 +5,7 @@ import ReallyDeleteTeam from '.'
 import {deleteTeamWaitingKey} from '../../constants/teams'
 import {anyWaiting} from '../../constants/waiting'
 
-type OwnProps = Container.RouteProps<
-  {
-    teamname: string
-  },
-  {}
->
+type OwnProps = Container.RouteProps< { teamname: string } >
```

Comment:
seems like `type OwnProps = Container.RouteProps<{teamname: string}>` is more our style?

Reply:
fixed tnx